xchange-cexio/src/main/java/info/bitrich/xchangestream/cexio/CexioStreamingExchange.java

```diff
@@ -40,17 +41,17 @@ public StreamingMarketDataService getStreamingMarketDataService() {
     }
 
     @Override
-    public void useCompressedMessages(boolean compressedMessages) {
-        throw new NotYetImplementedForExchangeException();
+    public StreamingPrivateDataService getStreamingPrivateDataService() {
+        return streamingPrivateDataService;
     }
 
-    public void setCredentials(String apiKey, String apiSecret) {
-        streamingOrderDataService.setApiKey(apiKey);
-        streamingOrderDataService.setApiSecret(apiSecret);
+    @Override
+    public void useCompressedMessages(boolean compressedMessages) {
+        streamingPrivateDataService.useCompressedMessages(compressedMessages);
     }
 
-    public CexioStreamingRawService getStreamingRawService() {
-        return streamingOrderDataService;
+    public void setCredentials(String apiKey, String apiSecret) {
```

Comment:
maybe we should just use the apiKey and apiSecret specified in the 'ExchangeSpecification'